frontend/src/metabase/sharing/components/SharingSidebar.jsx

```diff
@@ -106,6 +106,9 @@ const getEditingPulseWithDefaults = (state, props) => {
   if (!pulse.name) {
     pulse.name = dashboardWrapper.dashboards[dashboardWrapper.dashboardId].name;
   }
+  if (!pulse.dashboard_id) {
+    pulse.dashboard_id = dashboardWrapper.dashboards[dashboardWrapper.dashboardId].id;
+  }
```

Comment:
This comes back from the API directly, but I don't understand how the JSON response (with `dashboard_id` gets turned into the `pulse` object (which no longer has a `dashboard_id`). @kdoh any insights?

Reply:
I know before I added `dashboard_id` to Pulse, it was just looking at Pulse -> PulseCard -> DashboardCard -> Dashboard to get the dashboard ID for the Pulse

Reply:
@tsmacdonald Not sure. Off the cuff I would imagine that it might have something to do with how the entity system is putting it in the store, but in general I believe we just put everything from the api response object in.